{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2020-24584",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "An issue was discovered in Django 2.2 before 2.2.16, 3.0 before 3.0.10, and 3.1 before 3.1.1 (when Python 3.7+ is used). The intermediate-level directories of the filesystem cache had the system's standard umask rather than 0o077.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2020-24584",
				"url": "https://www.suse.com/security/cve/CVE-2020-24584"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1175784 for CVE-2020-24584",
				"url": "https://bugzilla.suse.com/1175784"
			}
		],
		"title": "SUSE CVE CVE-2020-24584",
		"tracking": {
			"current_release_date": "2024-07-20T03:46:25Z",
			"generator": {
				"date": "2023-02-15T03:54:23Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2020-24584",
			"initial_release_date": "2023-02-15T03:54:23Z",
			"revision_history": [
				{
					"date": "2023-02-15T03:54:23Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2024-07-20T03:46:25Z",
					"number": "3",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "HPE Helion OpenStack 8",
								"product": {
									"name": "HPE Helion OpenStack 8",
									"product_id": "HPE Helion OpenStack 8",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:hpe-helion-openstack:8"
									}
								}
							},
							{
								"category": "product_name",
								"name": "HPE Helion OpenStack Cloud 8",
								"product": {
									"name": "HPE Helion OpenStack Cloud 8",
									"product_id": "HPE Helion OpenStack Cloud 8"
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 5",
								"product": {
									"name": "SUSE Enterprise Storage 5",
									"product_id": "SUSE Enterprise Storage 5",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:5"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 7",
								"product": {
									"name": "SUSE OpenStack Cloud 7",
									"product_id": "SUSE OpenStack Cloud 7",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:7"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 8",
								"product": {
									"name": "SUSE OpenStack Cloud 8",
									"product_id": "SUSE OpenStack Cloud 8",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:8"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 9",
								"product": {
									"name": "SUSE OpenStack Cloud 9",
									"product_id": "SUSE OpenStack Cloud 9",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:9"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud Crowbar 8",
								"product": {
									"name": "SUSE OpenStack Cloud Crowbar 8",
									"product_id": "SUSE OpenStack Cloud Crowbar 8",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud-crowbar:8"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud Crowbar 9",
								"product": {
									"name": "SUSE OpenStack Cloud Crowbar 9",
									"product_id": "SUSE OpenStack Cloud Crowbar 9",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud-crowbar:9"
									}
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django",
								"product": {
									"name": "python-Django",
									"product_id": "python-Django",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django1",
								"product": {
									"name": "python-Django1",
									"product_id": "python-Django1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django-4.2.11-2.1",
								"product": {
									"name": "python310-Django-4.2.11-2.1",
									"product_id": "python310-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.14-1.1",
								"product": {
									"name": "python310-Django4-4.2.14-1.1",
									"product_id": "python310-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django-4.2.11-2.1",
								"product": {
									"name": "python311-Django-4.2.11-2.1",
									"product_id": "python311-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.14-1.1",
								"product": {
									"name": "python311-Django4-4.2.14-1.1",
									"product_id": "python311-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django-4.2.11-2.1",
								"product": {
									"name": "python312-Django-4.2.11-2.1",
									"product_id": "python312-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.14-1.1",
								"product": {
									"name": "python312-Django4-4.2.14-1.1",
									"product_id": "python312-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python36-Django-3.2.7-2.3",
								"product": {
									"name": "python36-Django-3.2.7-2.3",
									"product_id": "python36-Django-3.2.7-2.3"
								}
							},
							{
								"category": "product_version",
								"name": "python38-Django-3.2.7-2.3",
								"product": {
									"name": "python38-Django-3.2.7-2.3",
									"product_id": "python38-Django-3.2.7-2.3",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:3.2.7:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python39-Django-3.2.7-2.3",
								"product": {
									"name": "python39-Django-3.2.7-2.3",
									"product_id": "python39-Django-3.2.7-2.3",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:3.2.7:*:*:*:*:*:*:*"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django-4.2.11-2.1"
				},
				"product_reference": "python310-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.14-1.1"
				},
				"product_reference": "python310-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django-4.2.11-2.1"
				},
				"product_reference": "python311-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.14-1.1"
				},
				"product_reference": "python311-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django-4.2.11-2.1"
				},
				"product_reference": "python312-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.14-1.1"
				},
				"product_reference": "python312-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python36-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python36-Django-3.2.7-2.3"
				},
				"product_reference": "python36-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python38-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python38-Django-3.2.7-2.3"
				},
				"product_reference": "python38-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python39-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
				},
				"product_reference": "python39-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of HPE Helion OpenStack 8",
					"product_id": "HPE Helion OpenStack 8:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "HPE Helion OpenStack 8"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of HPE Helion OpenStack Cloud 8",
					"product_id": "HPE Helion OpenStack Cloud 8:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "HPE Helion OpenStack Cloud 8"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Enterprise Storage 5",
					"product_id": "SUSE Enterprise Storage 5:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Enterprise Storage 5"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE OpenStack Cloud 8",
					"product_id": "SUSE OpenStack Cloud 8:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE OpenStack Cloud 8"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django1 as component of SUSE OpenStack Cloud 9",
					"product_id": "SUSE OpenStack Cloud 9:python-Django1"
				},
				"product_reference": "python-Django1",
				"relates_to_product_reference": "SUSE OpenStack Cloud 9"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE OpenStack Cloud Crowbar 8",
					"product_id": "SUSE OpenStack Cloud Crowbar 8:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE OpenStack Cloud Crowbar 8"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django1 as component of SUSE OpenStack Cloud Crowbar 9",
					"product_id": "SUSE OpenStack Cloud Crowbar 9:python-Django1"
				},
				"product_reference": "python-Django1",
				"relates_to_product_reference": "SUSE OpenStack Cloud Crowbar 9"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2020-24584",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-24584"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in Django 2.2 before 2.2.16, 3.0 before 3.0.10, and 3.1 before 3.1.1 (when Python 3.7+ is used). The intermediate-level directories of the filesystem cache had the system's standard umask rather than 0o077.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"known_not_affected": [
					"HPE Helion OpenStack 8:python-Django",
					"HPE Helion OpenStack Cloud 8:python-Django",
					"SUSE Enterprise Storage 5:python-Django",
					"SUSE OpenStack Cloud 7:python-Django",
					"SUSE OpenStack Cloud 8:python-Django",
					"SUSE OpenStack Cloud 9:python-Django1",
					"SUSE OpenStack Cloud Crowbar 8:python-Django",
					"SUSE OpenStack Cloud Crowbar 9:python-Django1"
				],
				"recommended": [
					"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
					"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
					"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
					"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
					"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
					"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-24584",
					"url": "https://www.suse.com/security/cve/CVE-2020-24584"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1175784 for CVE-2020-24584",
					"url": "https://bugzilla.suse.com/1175784"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2020-08-26T11:51:37Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-24584"
		}
	]
}
